tools/deployment/presto-clp/coordinator/config-template/metadata-filter.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
@anlowee I think we should explain to the user how to configure this for the timestamp field in their logs, right? And also that it may need to be different for each dataset they compress.